turbinia/client.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
```

Comment:
Drop the shebang. We will not execute this directly.

Reply:
Done.